src/components/RangeSlider/icons/icn-decrease-btn.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export default () => (
+  <svg
+    width="25px"
+    height="25px"
+    viewBox="0 0 32 32"
+  >
+    <g
+      fill="none"
+      stroke="none"
+      strokeWidth="1"
+      fillRule="evenodd"
+    >
+      <g transform="translate(-52.000000, -822.000000)">
```

Comment:
Maybe this svg can still be cleaned up a little bit, I don't think the two `g` tags are required (however, you'd need to check positioning then)

Reply:
done